tests/fuzz/fuzz-bech32.c

```diff
@@ -60,5 +60,18 @@ void run(const uint8_t *data, size_t size)
 		assert(memcmp(data_out, data, data_out_len) == 0);
 	}
 
+	/* Test 8-to-5 bit roundtrip conversion */
+	uint8_t *converted = tal_arr(tmpctx, uint8_t, size * 2);
+	size_t converted_len = 0;
+	if (bech32_convert_bits(converted, &converted_len, 5, data, size, 8, 1)) {
+		uint8_t *deconverted = tal_arr(tmpctx, uint8_t, converted_len * 2);
```

Comment:
Does `deconverted` need to be size `converted_len * 2`?  If we're going from 5 bits back to 8 bits, shouldn't we need at most `size` bytes?